src/store.ts

```diff
@@ -25,7 +25,7 @@ export class Store {
 
   transactionManager: TransactionsManager;
 
-  callbacks: TransactionCallback[] = [];
+  private callbacks: TransactionCallback[] = [];
```

Comment:
lets use the actually private properties via `#`

Reply:
for everything that isn't supposed to be public

Reply:
Need to tweak ts config. Let's do this later.